db/migrate/20160605140228_create_searches.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSearches < ActiveRecord::Migration
```

Comment:
If we could post-date the filename timestamp to fall after existing migrations, that'd be excellent; we may have to do this just before final merge in case other migrations are merged first.